types/iyzipay/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
```

Comment:
why you need `dom` library? (I could miss used symbol in the index file, sorry)

Reply:
I have removed commonjs, instead I used es6. It seems I looks like I dont need commonjs. I commited the latest changes